src/types/video.ts

```diff
@@ -40,6 +40,15 @@ export type ReactVideoSourceProperties = {
   textTracks?: TextTracks;
 };
 
+export type TPlaybackStatus = {
```

Comment:
PlaybackStatus would be enough ?

Reply:
I think it's better to add the prefix `T` before type names, meaning playbackStatus would be a type. Similarly, we can add an `I` before interface names
FX: type TSample = {} and interface ISample = {}
WDYT?

Reply:
As Robert Martin wrote in his "Clean Code" - T, I, Enum, List (and so on) suffixes prefixes/suffixes before types, interfaces, enums (and so on) are meaningless.

We are not necessarily need to follow what he writes in his book, but it's better to follow guidelines that are accepted in the project you are working on (for example, as I can see in this project suffixes/prefixes are not used)